app/controllers/projects_controller.rb

```diff
@@ -0,0 +1,6 @@
+class ProjectsController < ApplicationController
+  def show
+    @project = Project.find(params[:id])
+    @contestants = @project.contestants
```

Comment:
This second contestants variable is not strictly necessary because you could, in the view, just do `@project.contestants`